web/bin/serve_more

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+#  serve more
+#  Script for making it easy that shoelaces
+#  can serve more static content.
+#  That content came to various way to the shoelaces server
+#  ( downloads, Linux distro packages, self build )
+#  This script creates symbolic links to it.
+#
+#  Start like `bin/serve_more  <file>...`.
+
+if [ ${#} -lt 1 ] ; then
+	echo "E: No datafiles provided"
+	echo "I: Add filenames of text files with lines like"
+	echo "I:   netboot/foo/vmlinux  /srv/downloaded_foo/linuz_6.1"
+	exit 1
+fi
+
+TOP=${PWD}  # Top of web directory, the 'static-dir=' value
+
+while read TOSERVE CONTENT
+do
+	echo ${TOSERVE} ${CONTENT}
+	TSDIR=$( dirname ${TOSERVE} )
+	TSFN=$( basename ${TOSERVE} )
+	cd ${TOP}
```

Comment:
In scripts, it's preferable to use `pushd` and `popd`, a feature known as the [Directory Stack](https://www.gnu.org/software/bash/manual/html_node/The-Directory-Stack.html). That way, you won't surprise a user by inadvertently changing directories.

Anyway, I think that in this situation you can skip changing directories and use absolute paths in `ln`.